packages/story-editor/src/components/library/panes/pageTemplates/defaultTemplates.js

```diff
@@ -147,16 +171,19 @@ function DefaultTemplates({ pageSize }) {
             {__('Templates', 'web-stories')}
           </Headline>
         </ActionRow>
-        {!isLoading && pageTemplatesParentRef.current ? (
+
+        {isLoading || !pageTemplatesParentRef.current ? (
+          <LoadingContainer>
+            <LoadingSpinner animationSize={64} numCircles={8} />
+          </LoadingContainer>
+        ) : pageTemplates.length === 0 ? (
+          <Message>{__('No templates found.', 'web-stories')}</Message>
```

Comment:
```suggestion
          <Message>{__('No page templates found.', 'web-stories')}</Message>
```

`No page templates found.` is a translation string that already exists